src/get_address.c

```diff
@@ -9,10 +9,40 @@
 #include "os.h"
 #include "utils.h"
 #include "ux.h"
+#include "menu.h"
+
+#ifdef HAVE_NBGL
+#include "nbgl_use_case.h"
+#endif
 
 static char address[FULL_ADDRESS_LENGTH];
 
-static uint8_t set_result_get_address();
+static uint8_t set_result_get_address(void) {
+    uint8_t tx = 0;
+    uint8_t address_size = strlen(address);
+
+    G_io_apdu_buffer[tx++] = address_size;
+    memmove(G_io_apdu_buffer + tx, address, address_size);
+    tx += address_size;
+
+    return tx;
+}
+
+#if defined(TARGET_STAX)
+
+static void callback_match(bool match) {
+    if (match) {
+        send_response(set_result_get_address(), true, true);
```

Comment:
There should be a `nbgl_useCaseStatus("ADDRESS\nVERIFIED", true, ui_idle);` here?

Reply:
Indeed !